docs/source/models/intervention_models/intrapartum/azithromycin_intervention.rst

```diff
@@ -0,0 +1,265 @@
+- We have excluded the effect of azithromycin on pneumonia incidence/mortality, because this cause is currently lumped with 'other causes'.
+- We currenty do not model the impact of intrapartum azithromycin on the incidence of preterm births, despite *some* literature
+  evidence that suggests there may be a significant impact. For reference, this `2021 systematic review <https://pmc.ncbi.nlm.nih.gov/articles/PMC8436060/>`_
+  found an RR of 0.79 (95% CI 0.68-0.93) for LBW and an RR of 0.87 (95% CI 0.78-0.98) for premature births. They also reported an 
+  increase in stillbirht incidence. However, more recent publications (the 2024 review referenced above and  `this 2022 paper <https://link.springer.com/article/10.1007/s40261-022-01203-0>`_) 
+  have reported that there is no conclusive evidence to support that azithromycin use by pregnant women causes adverse 
+  neonatal outcomes.
+- Baseline coverage data for azithromycin in CEmONC and BEmONC is only reflective of Ethiopian health systems in 2015-2018. These placeholder values come 
+  from three data sources, all for Ethiopia, all identified by the Health Systems team at IHME: the 2016 Ethiopia EmONC Final 
+  Report found 39.2% of BEmONC facilities and 90.4% of CEmONC facilities have azithromycin; the 2016 SARA Report 
+  found 68.4% of BEmONC facilities and 90.5% of CEmONC facilities have azithromycin; the 2018 SARA Report found 75.9% of BEmONC facilities
+  and 98.0% of CEmONC facilities have azithromycin. While we plan a data strategy to fill the gaps we have used a simple average.
+- We assume that baseline coverage for azithromycin in home births is 5% (this is not data-backed).
+
+.. todo::
+
+  If more suitable baseline coverage data for azithromycin for maternal sepsis at all facility types for Nigeria and Pakistan, we should use that data instead and update 
+  this documentation accordingly. We also need to decide if/how we would model the effect of intrapartum azithromycin on preterm incidence. 
+
+Validation and Verification Criteria
+------------------------------------
+
+- Population-level incidence rate should be the same as when this intervention is not included in the model
+- The ratio of maternal sepsis incidence among those without azithromycin access divided by those with azithromycin access 
+  should equal the relative risk parameter used in the model
+- The baseline coverage of azithromycin in each facility type should match the values in the artifact
+- Validation: how does the sepsis incidence rate in a counterfactual scenario with 100% antibiotic access compare to sepsis incidence rates in high-income countries?  They should be close, and the counterfactual should not be lower.
```

Comment:
Hmm, I am not convinced of this... there are likely other things that influence sepsis incidence rates (like delivery location sanitation). It could be interesting to check, but I'm not sure this should be the expectation